_data/books.yml

```
@@ -0,0 +1,25 @@
+- index: 1 # mentor id 1
+  books:
+    - title: "Thinking, Fast and Slow by Daniel Kahneman"
+      comments:
+    - title: "Never Split the Difference: Negotiating as if Your Life Depended on It  by Chris Voss and Tahl Raz"
+      comments:
+    - title: "The Culture Map - Breaking Through the Invisible Boundaries of Global Business by Erin Meyer"
+      comments:
+    - title: "The Pragmatic Programmer: From Journeyman to Master by Andy Hunt and Dave Thomas"
+      comments:
+
+- index: 7 # mentor id 7
+  books:
+    - title: "Clean Code: A Handbook of Agile Software Craftsmanship by Robert C. Martin"
```

What I mentioned was that if you don't have any characters to break the syntax you can just add the text. 

In this case test you have this : you can solve with |
```suggestion
    - title: |
    Clean Code: A Handbook of Agile Software Craftsmanship by Robert C. Martin"
```